docs/install/ipfs-desktop.md

```diff
@@ -92,6 +94,14 @@ While these instructions are specific to Ubuntu, they will likely work with most
 
    ![IPFS icon shown in the Ubuntu status bar.](./images/ipfs-desktop/install-ubuntu-ipfs-running-status-bar.png)
 
+## Package Managers
+
+- **Homebrew** - `brew cask install ipfs`
+- **Chocolatey** - `choco install ipfs-desktop`
+- **Scoop** - `scoop install ipfs-desktop`
+- **Snap** - `snap install ipfs-desktop`
+- **AUR** - [`ipfs-desktop` package](https://aur.archlinux.org/packages/ipfs-desktop/) maintained by [@alexhenrie](https://github.com/alexhenrie)
```

Comment:
Can we convert this into a table and add links to each package-managers page? Similar to what you've done with AUR.

Reply:
@johnnymatthews 
![image](https://user-images.githubusercontent.com/15967809/96488878-4e025900-125c-11eb-8530-1bbc8a04cd30.png)

it looks like this now and below are the links that are given

| Package Manager                                                                                                    | Command                      |
| ------------------------------------------------------------------------------------------------------------------ | ---------------------------- |
| [Homebrew](https://formulae.brew.sh/cask/ipfs/)                                                                    | `brew cask install ipfs`     |
| [Chocolatey](https://chocolatey.org/packages/ipfs-desktop)                                                         | `choco install ipfs-desktop` |
| [Scoop](https://github.com/lukesampson/scoop-extras/blob/master/bucket/ipfs-desktop.json)                          | `scoop install ipfs-desktop` |
| [Snap](https://snapcraft.io/ipfs-desktop)                                                                          | `snap install ipfs-desktop`  |
| [AUR](https://aur.archlinux.org/packages/ipfs-desktop/) maintained by [@alexhenrie](https://github.com/alexhenrie) | `ipfs-desktop`               |

Reply:
Excellent. Looks good!